examples/StandardFirmataEthernet/StandardFirmataEthernet.ino

```diff
@@ -939,7 +944,11 @@ void loop()
 #if !defined local_ip && !defined YUN_ETHERNET
   // only necessary when using DHCP, ensures local IP is updated appropriately if it changes
   if (Ethernet.maintain()) {
+    #if defined remote_ip || defined remote_host
     stream.maintain(Ethernet.localIP());
+    #else
+    stream.maintain();
```

Comment:
I don't think you need to call stream.maintain() here since it is already called on each iteration of loop when firmata.available is called. However, you may need to copy [the maintain(IPAddress local) method](https://github.com/firmata/arduino/blob/master/utility/EthernetClientStream.h#L101) from EthernetClientStream to EthernetServerStream and then leave this block of code as it originally was. You should test this however to ensure there are not side effects.